src/Middleware/ExecutionChain.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace League\Route\Middleware;
+
+use \Psr\Http\Message\ResponseInterface;
```

Comment:
Yeah, just an oversight that one

On Wed, 6 Apr 2016 09:14 Frank de Jonge, <notifications@github.com> wrote:

> In src/Middleware/ExecutionChain.php
> <https://github.com/thephpleague/route/pull/115#discussion_r58666937>:
>
> > @@ -0,0 +1,47 @@
> > +<?php
> > +
> > +namespace League\Route\Middleware;
> > +
> > +use \Psr\Http\Message\ResponseInterface;
>
> Use statement are always from global and \ prefixes are troublesome is
> more exotic php versions/envs. I'd recommend removing it, I've dealt with
> this in the past and it wasn't fun.
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/thephpleague/route/pull/115/files/bfab6c47e06c6ed5e2c864f019359eef9649570e#r58666937>
>